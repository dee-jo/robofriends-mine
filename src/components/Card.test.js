import { shallow, mount, render } from 'enzyme';
import Card from './Card';
import React from 'react';

test('Expect to render Card component', () => {
  expect(shallow(<Card />).debug()).toMatchSnapshot();
})