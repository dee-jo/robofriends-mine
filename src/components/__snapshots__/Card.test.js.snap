// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Card component 1`] = `
"<div className=\\"tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5\\">
  <img src=\\"https://robohash.org/undefined?size=200x200\\" alt=\\"robots\\" />
  <div>
    <h2 />
    <p />
  </div>
</div>"
`;

// CODE COVERAGE
// npm test -- --coverage --watchAll=false 